.github/workflows/build_publish_docs.yml

```
@@ -0,0 +1,49 @@
+
+      - name: Install dependencies
+        run: cd documentation && yarn install --frozen-lockfile
+
+      - name: Build website
+        run: cd documentation && yarn build
+        env:
+          DOCS_BASE_URL: ${{ vars.DOCS_BASE_URL }}
+          DOCS_URL: ${{ vars.DOCS_URL }}
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)